core/vtk/ttkAlgorithm/ttkAlgorithm.h

```diff
@@ -2,7 +2,6 @@
 /// \brief The Topology ToolKit - VTK wrapping code for the processing
 /// packages.
 /// @{
-/// \ingroup vtk
```

Comment:
why removing this?

Reply:
this should be in the `vtk` category, shouldn't it?

Reply:
It already is: see the `\defgroup vtk` at the top of the file.

Reply:
The corresponding Doxygen warning:
> Parsing file /home/pierre/Projects/ttk/core/vtk/ttkTria/home/pierre/Projects/ttk/core/vtk/ttkAlgorithm/ttkAlgorithm.h:1: warning: Refusing to add group vtk to itself

Reply:
yes, nice.